packages/generator-langium/langium-template-vscode/.package.json

```diff
@@ -1,4 +1,7 @@
 {
+    "name": "<%= extension-name %>",
```

Comment:
I wasn't specific enough here: `name` should be in the core template because it's used by npm as well.